node/src/index.ts

```diff
@@ -815,6 +817,7 @@ export class LocalTable<T = number[]> implements Table<T> {
   /**
    * Clean up old versions of the table, freeing disk space.
    *
+   * Note: this API is not yet available on LanceDB Cloud
```

Comment:
i think all of these repeated notes will make the docs look really messy. And chances are when we implement some of these in cloud we'll forget to update these.